samples/bluetooth/mesh/light_switch/lpn.conf

```
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BT_MESH_LOW_POWER=y
+CONFIG_BT_DEVICE_NAME="Mesh Light Switch LPN"
+CONFIG_BT_MESH_RELAY=n
+CONFIG_BT_MESH_FRIEND=n
+
+# Power saving configuration:
+
+# Disable serial communication
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+
+# Disable secure beacons
+CONFIG_BT_MESH_BEACON_ENABLED=n
+
+# Extend LPN poll interval to 1 minute
+CONFIG_BT_MESH_LPN_POLL_TIMEOUT=600
```

```suggestion
CONFIG_BT_MESH_LPN_POLL_TIMEOUT=600
CONFIG_BT_MESH_GATT_PROXY=n
```

because the sample's PRJ conf enables the GATT Proxy. We want only 'BT_MESH_GATT_SERVER' (which gets automatically selected because of CONFIG_BT_MESH_PB_GATT=y).

---

See below comment ⬇️

---

Ok.